### PR TITLE
Firefighting cabinet

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary
Content "Firefighting cabinets for decoration and looting within large buildings."

#### Purpose of change

Some extra verisimilitude for large industrial and commercial complexes

#### Describe the solution

In the case the name isnt familiar fireman cabinets are these things: 
image

Most of the time they just have large/small extinguishers, but they can also spawn with a hose and rarely a fireaxe.

#### Alternatives you have considered
Maybe a different name?

#### Testing

Load game, check some cabinets for stuff.

### CODE REVIEW

File: data/json/furniture_and_terrain/furniture-storage.json [hunk 1]
```diff
@@ -994,6 +994,79 @@
...
+    "looks_like": "f_displaycase",
+    "description": "A conspicuously painted metallic cabinet, visibly holding tools behind a thin panel of tempered glass. ",
+    "symbol": "#",
+    "color": "light_red",
+    "move_cost_mod": -1,
+    "coverage": 80,
+    "required_str": -1,
+    "flags": [ "TRANSPARENT", "SEALED", "PLACE_ITEM", "PICKABLE" ],
+    "lockpick_result": "f_fireman_cabinet_o",
+    "lockpick_message": "With a satisfying click, the lock and the cabinet open.",
```
Comment (Night-Pryanik) [inline data/json/furniture_and_terrain/furniture-storage.json:1010]:
```suggestion
    "lockpick_message": "With a satisfying click, the lock on the cabinet opens.",
```

File: data/json/furniture_and_terrain/furniture-storage.json [hunk 2]
```diff
@@ -994,6 +994,79 @@
...
+      "furn_set": "f_fireman_cabinet_b",
+      "items": [ { "item": "glass_shard", "count": [ 25, 50 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_fireman_cabinet_b",
+    "name": "open fireman's cabinet",
+    "looks_like": "f_displaycase_b",
+    "description": "A conspicuously painted metallic cabinet.  It's protective glass panel lies broken from some past emergency or looting attempt.",
```
Comment (Night-Pryanik) [inline data/json/furniture_and_terrain/furniture-storage.json:1051]:
```suggestion
    "description": "A conspicuously painted metallic cabinet.  Its protective glass panel lies broken from some past emergency or looting attempt.",
```

**Comment by Night-Pryanik** [comment]
Do you plan to add them to other locations as well?

File: data/json/furniture_and_terrain/furniture-storage.json (hunk 2 above)
Comment (actual-nh) [reply data/json/furniture_and_terrain/furniture-storage.json:1051]:
```suggestion
    "description": "A metallic cabinet painted to stand out.  Its protective glass panel lies broken from some past emergency or looting attempt.",
```

File: data/json/furniture_and_terrain/furniture-storage.json [hunk 3]
```diff
@@ -994,6 +994,79 @@
...
+      "furn_set": "f_fireman_cabinet_b",
+      "items": [ { "item": "glass_shard", "count": [ 25, 50 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_fireman_cabinet_o",
+    "name": "unlocked fireman's cabinet",
+    "looks_like": "f_displaycase",
+    "description": "A conspicuously painted metallic cabinet.  Its glass panel door was left unlocked and slightly ajar.",
```
Comment (John-Candlebury) [inline data/json/furniture_and_terrain/furniture-storage.json:1028]:
```suggestion
    "description": "A metallic cabinet painted to stand out.  Its glass panel door was left unlocked and slightly ajar.",
```

**Comment by John-Candlebury** [comment]
> 
> 
> Do you plan to add them to other locations as well?

Didnt originally plan to do this in here.

**Pull request merged**